utils/config_manager.py

```
@@ -795,6 +874,145 @@ def ensure_chara_directory(self):
+        if default_value is None:
+            default_value = self.build_default_character_tombstones_state()
+        return self._load_json_file(self.character_tombstones_state_path, default_value)
+
+    def save_character_tombstones_state(self, data):
+        """保存角色 tombstone 本地状态。"""
+        if not self.ensure_local_state_directory():
+            raise OSError("Failed to ensure local state directory before saving character_tombstones_state")
+        self._save_json_file(self.character_tombstones_state_path, data)
+
+    def ensure_cloudsave_state_files(self):
+        """确保本地 cloudsave 相关状态文件存在，返回是否发生创建。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)